media-plugins/vidstab/files/vidstab-gentoo.diff

```diff
@@ -0,0 +1,52 @@
+--- a/CMakeLists.txt    2017-06-18 00:17:02.883555561 +0200
```

Comment:
Let's make this patch suitable for upstream inclusion, then you should submit it. Might be better to split it into separate issues, though.